Samples/kotlin/src/main/java/com/phunware/kotlin/sample/OneWayRoutingActivity.kt

```diff
@@ -112,7 +117,7 @@ class OneWayRoutingActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
 
         // Create the map manager and fragment used to load the building
         mapManager = PhunwareMapManager.create(this)
-        val mapFragment = fragmentManager.findFragmentById(R.id.map) as MapFragment
+        val mapFragment = supportFragmentManager.findFragmentById(R.id.map) as SupportMapFragment
```

Comment:
We should never being using Platform/Native fragments.  Where possible we and our customers should only be using Support Fragments.  Thankfully we provide a "support" version of our MapFragment -- so I am using this here.